tests/cross/src/main/scala/tests/pc/BaseTypeDefinitionSuite.scala

```diff
@@ -0,0 +1,65 @@
+package tests.pc
+
+import tests.BasePCSuite
+import scala.concurrent.duration.Duration
+import scala.meta.internal.metals.CompilerOffsetParams
+import collection.JavaConverters._
+import org.eclipse.{lsp4j => l}
+
+class BaseTypeDefinitionSuite extends BasePCSuite {
+
+  override def beforeAll(): Unit = {
+    indexJDK()
+    indexScalaLibrary()
+  }
+
+  def locationFromString(str: String): l.Location = {
+    def positionFromString(str: String): l.Position = {
+      val ints = str.split(":").map(_.toInt)
+      new l.Position(ints.head, ints.last)
+    }
+
+    val split = str.split("""/\\""")
+    assertEquals(split.length, 3, "Incorrect location string format")
```

Comment:
print the actual content and maybe a hint on why it is incorrent

Reply:
Fixed.